.github/workflows/release.yml

```diff
@@ -10,19 +10,22 @@ permissions:
   contents: write
 
 env:
-  PREVIEW_TASK: publishMavenPublicationToSonatypeSnapshotRepository
-  PUBLISH_TASK: publishMavenPublicationToSonatypeRepository
+  PREVIEW_TASK: publishToSonatype
+  PUBLISH_TASK: publishToSonatype closeSonatypeStagingRepository
```

Comment:
```suggestion
  PUBLISH_TASK: publishToSonatype closeAndReleaseSonatypeStagingRepository
```

Reply:
I'd be more comfortable if we didn't automate the 'release' step just yet to validate that nothing is broken with the next release. Thoughts?

Reply:
oh yeah, I forgot about that comment you made earlier. disregard this one, we can automate the final step once we have vetted all of this.